web/app/scripts/src/directives/form.es.js

```diff
@@ -30,6 +30,17 @@ module.directive('surveysForm', [
             templateUrl: 'templates/incuna-surveys/form/survey-form.html',
             link: function (scope) {
                 scope.model = {};
+                scope.percentageComplete = 0 + '%';
+                let totalQuestionCount = 0;
+
+                const countQuestionsTotal = function () {
+                    const questions = scope.form.fieldsets;
+
+                    questions.forEach((question) => {
```

Comment:
Could this use [reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce)  `questions.reduce((value, questions) => {value + question.fields.length} , 0)`